lib/services/filters.js

```diff
@@ -16,13 +16,31 @@ function filterRefs(op) {
   return op;
 }
 
+/**
+ * Generic function for testing if op is of
+ * a certain type. Just call the function with what
+ * you're looking for and then you have a filter function.
+ *
+ * - Pages? isOpFor('pages')
+ * - User? isOpFor('user')
+ * - Component Foo? isOpFor('foo')
```

Comment:
Would this actually work? Note your trailing slash below

Reply:
Yes, it would. `.../components/foo/instances/...`

Reply:
Ah, thought you meant instance name